src/pat/tinymce/tinymce--implementation.js

```diff
@@ -134,6 +134,7 @@ export default class TinyMCE {
         if (lang !== "en" && self.options.tiny.language !== "en") {
```

Comment:
Shouldn't this condition be an OR or AND instead?

Because the value of self.options.tiny.language by default is "en".

It's not working for me in pt_BR language. 

After changing to OR it works.